hive-agent/src/main/java/org/apache/ranger/authorization/hive/authorizer/RangerHiveAuthorizer.java

```diff
@@ -2630,7 +2627,7 @@ private List<HivePrivilegeInfo> getHivePrivilegeInfos(HivePrincipal principal, H
 
 	private RangerPolicy getRangerPolicy(Map<String, RangerResourceACLs.AccessResult> accessResults, String rangerACL){
 		RangerPolicy ret = null;
-		if (MapUtils.isNotEmpty(accessResults)) {
```

Comment:
why was this changed?  MapUtils is still here isn't it?

Reply:
`MapUtils` is here but `isNotEmpty()` doesn't exist.